data_processors/pipeline/lambdas/fastq.py

```diff
@@ -0,0 +1,166 @@
+try:
+    import unzip_requirements
```

Comment:
Yap; this is mainly for Serverless Framework environment; the `serverless-python-requirements` plugin, in regard to _Dealing with Lambda's size limitations_(1) for unzip requirements/dependencies(2). Otherwise, pass as not needed.

(1): https://www.serverless.com/plugins/serverless-python-requirements
(2): https://github.com/UnitedIncome/serverless-python-requirements/blob/master/unzip_requirements.py